src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -47,35 +48,22 @@
 }
 
 
-class DeploymentOption:
-    def __init__(self, name: str, available=False, capacity=0, used=0, hdd_used=0,
-                 ssd_used=0, nvme_used=0):
-        self.name = name
-        self.available = available
-        self.capacity = capacity
-        self.used = used
-        self.hdd_used = hdd_used
-        self.ssd_used = ssd_used
-        self.nvme_used = nvme_used
-
-    def as_dict(self):
-        return {
-            'name': self.name,
-            'available': self.available,
-            'capacity': self.capacity,
-            'used': self.used,
-            'hdd_used': self.hdd_used,
-            'ssd_used': self.ssd_used,
-            'nvme_used': self.nvme_used
-        }
-
-
 class DeploymentOptions:
     def __init__(self):
         self.options = {
-            'cost-capacity': DeploymentOption('cost-capacity'),
-            'throughput': DeploymentOption('throughput-optimized'),
-            'iops': DeploymentOption('iops-optimized'),
+            OsdDeploymentOptions.COST_CAPACITY:
+                HostStorageSummary(OsdDeploymentOptions.COST_CAPACITY,
+                                   title='Cost/Capacity-optimized',
+                                   desc='All the available HDDs are selected'),
+            OsdDeploymentOptions.THROUGHPUT:
+                HostStorageSummary(OsdDeploymentOptions.THROUGHPUT,
+                                   title='Throughput-optimized',
+                                   desc="""HDDs/SSDs are selected for data
+                                   devices and SSDs/NVMes for DB/WAL devices"""),
```

Comment:
Otherwise that string will have a lot of padding:
```py
HDDs/SSDs are selected for data
                                   devices and SSDs/NVMes for DB/WAL devices
```
```suggestion
                                   desc="HDDs/SSDs are selected for data"
                                   "devices and SSDs/NVMes for DB/WAL devices"),
```

Reply:
done